content/en/docs/ibm/using-icr.md

```diff
@@ -1,24 +1,22 @@
 +++
 title = "Using IBM Cloud Container Registry (ICR)"
-description = "Instructions for using private images from ICR"
+description = "This guide covers certain scenarios involving using private container images with the IBM Cloud Container Registry (ICR)."
 weight = 15
 +++
 
-This guide describe steps for use cases involving private container images by using IBM Cloud Container Registry (ICR).
-
 ## Prerequisites
 
 * Install and configure the [IBM Cloud CLI](https://cloud.ibm.com/docs/cli?topic=cli-getting-started).
 * Install the CLI plug-in for the IBM Cloud Container Registry by running the command `ibmcloud plugin install container-registry`.
-* Create a namespace in ICR with command `ibmcloud cr namespace-add <my_namespace>`, replace `<my_namespace>` with your preferred name.
+* Create a namespace in ICR with the command `ibmcloud cr namespace-add <my_namespace>`, replace `<my_namespace>` with your preferred name.
 
-**Note**: the [ICR namespace](https://cloud.ibm.com/docs/Registry?topic=Registry-getting-started#gs_registry_namespace_add) is different from the Kubeflow Profile namespace. The ICR namespace is used to group container images stored in ICR while a [Kubeflow Profile](/docs/components/multi-tenancy/overview/) namespace is a grouping of all Kubernetes clusters onwed by a user.
+**Note**: the [ICR namespace](https://cloud.ibm.com/docs/Registry?topic=Registry-getting-started#gs_registry_namespace_add) is different from the Kubeflow Profile namespace. The ICR namespace is a group container images stored in ICR while a [Kubeflow Profile](/docs/components/multi-tenancy/overview/) namespace is a group of all Kubernetes clusters owned by a user.
 
 ## Image pull secret
 
-An image pull secret is required to to pull container images from IBM Cloud Container Registry. You can use the default image pull secret set up by the cluster or use your account's IAM API key.
+It needs an image pull secret to pull container images from IBM Cloud Container Registry. You can use the default image pull secret set up by the cluster or your account's IAM API key.
```

Comment:
You start a new section with "It" but there's no previous context. It's not clear what "it" means here. Was there something big that needed to be changed before? I understand there was a second "to" (in "to to" that just needed to be removed. Now the sentence is a bit "lost".

Proposal - keep the old with "to":

> An image pull secret is required to pull container images from IBM Cloud Container Registry.

Just making sure the readers don't get confused here.

Reply:
Also keep in mind @adrian555 's comment:

<img width="835" alt="image" src="https://user-images.githubusercontent.com/19637339/88596469-11f49900-d05d-11ea-8ed6-5445a3c7bd34.png">

Reply:
make sense, I added context so that I can keep "it". (The more I use http://www.hemingwayapp.com/ the less I want to use passive voice).